### DISCUSSION TITLE
Depth in navbar

### DISCUSSION THREAD

**Discussion by Gordolin** [razonyang/hugo-theme-bootstrap#841] [Q&A] [answered]
Hi, i need in docs a deeper navigation, but its ist no displayed in the navbar.
Example:
content
-docs
--sub1
--_index.md
---- sub2
----_index.md
------sub3
------_index.md

In the navigation bar i can find documents in level sub1.  sub2 and below ist not displayed. The docs can be found with the search of course. _index.md is in each subdir.

Is there a way to go a little bit deeper in navigation?

**Comment by razonyang** [comment, marked as answer]
The theme should support unlimited levels on the `docs` layout, just test this with the [skeleton](https://github.com/razonyang/hugo-theme-bootstrap-skeleton) template. Content structure and preview as follows:

```sh
$ tree content/docs
content/docs
├── _index.md
├── installation
│   ├── _index.md
│   ├── linux
│   │   ├── _index.md
│   │   ├── archlinux
│   │   │   └── index.md
│   │   ├── sub-linux
│   │   │   ├── _index.md
│   │   │   ├── foobar
│   │   │   │   └── index.md
│   │   │   └── go-deeper
│   │   │       ├── _index.md
│   │   │       └── leaf
│   │   │           └── index.md
│   │   └── ubuntu
│   │       └── index.md
│   └── windows
│       └── index.md
└── introduction
    └── index.md

11 directories, 11 files
```

![image](https://user-images.githubusercontent.com/17720932/210172473-264a6076-7c9b-4e9a-9cb3-9d0c4f9243b9.png)


Could you please provide the output of `tree content/docs` with better format?

> Sometime it may be a cache issue, restart the Hugo server and clear browser cache to make sure the page isn't cached.

**Reply by Gordolin** [reply]
Hi, i think it has been a browser chaching problem. After cleraing the browser cache ist worked as descibed.

**Comment by razonyang** [comment]
If you're talking about the top app bar menus, unfortunately, it's limited up to two levels.

![image](https://user-images.githubusercontent.com/17720932/210172785-ca3e8be6-a585-4375-95b1-9a529c39391e.png)
